doc/cookbook/source/examples/observers/least_angre_regression.rst

```diff
@@ -0,0 +1,50 @@
+=================================
+Observable Least Angle Regression
+=================================
+
+In this example, we will show how it is possible to obtain useful information from
+a Shogun's model while it is training. Here we will train a Least Angle
+Regression model. If you need more details about the LARS, have a look
+at its dedicated cookbook `example <../regression/least_angle_regression.html>`_.
+
+-------
+Example
+-------
+
+As usual, we first create the train and test features needed.
+
+.. sgexample:: least_angle_regression.sg:create_features
+
+Secondly, we do a round of preprocessing, by normalizing both the train and test features.
+
+.. sgexample:: least_angle_regression:preprocess_features
+
+We also create an instance of :sgclass:`CLeastAngleRegression` with some custom parameters
+which will be our model.
+
+.. sgexample:: least_angle_regression:create_instance
+
+Then we create an observer object which is attached to the LARS object. Here
+we are using the :sgclass:`CParameterObserverLogger` which basically will print
+on stdout all the information it will get from the machine during the training
```

Comment:
"which basically prints all information it gets from the machine during training on stdout"